snabble/Android-SDK
# ui-toolkit: Receipt list to display receipts in detail

Add a list and the functionality to display a list of all receipts and receipts in detail.

#### How to test
* Make at least one purchase
* Enter the receipt list from either
  * _Home_ -> _Previous purchases_: _More_
  * or _Profile_ -> _Last Purchases_
* Select a purchase to see the progress dialog, followed by the receipt in detail or an error in a SnackBar
- Alternatively select a purchase from _Home_ -> _Previous purchases_
